tools/cli/update-version.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const { execCommand, getAbsolutePath, replaceContentInFile, versionDiff } = require("./utils");
+const fs = require("fs");
+
+const args = process.argv.slice(2);
+
+// Default remote for the wso2 repository.
+let remote = "upstream";
+// Default update type.
+let updateType = "patch";
+
+if (args[0] && args[1]) {
+    remote = args[0];
+    updateType = args[1];
+} else if (args[0]) {
+    updateType = args[0];
+}
+
```

Comment:
Shall we print some error messages if the args are not in expected format?

Reply:
This is not needed, because args have default values, if they provided more args those will be neglected.